contentcuration/contentcuration/static/js/edit_channel/file_upload/views.js

```diff
@@ -79,7 +79,7 @@ var FileModalView = BaseViews.BaseModalView.extend({
     },
     close_file_uploader:function(event){
         var self = this;
-        if(this.file_upload_view.collection.length === 0 || this.file_upload_view.current_view.uploads_in_progress){
+        if(this.file_upload_view.collection.length === 0 && this.file_upload_view.current_view.uploads_in_progress > 0){
```

Comment:
When files are being uploaded, a user could easily exit the modal. This catches that and prompts them with "Are you sure you want to leave?"